sys-power/powertop/powertop-9999.ebuild

```diff
@@ -96,6 +96,9 @@ pkg_setup() {
 
 src_prepare() {
```

Comment:
write the `src_prepare` like so:

```
src_prepare() {
	default
	if [[ ${PV} == "9999" ]] ; then
		chmod +x scripts/version || die "Failed to make 'scripts/version' executable"
		scripts/version || die "Failed to extract version information"
		eautoreconf
	fi
}
```
This applies `PATCHES` whether `PV` is `9999` or not, and then calls `eautoreconf` in the live ebuild.